src/Model/Accommodation.php

```diff
@@ -94,21 +80,10 @@ public function getRentalUnits(): array
     }
 
     /**
-     * @param RentalUnit[] $rentalUnits
-     * @return self
-     */
-    public function setRentalUnits(array $rentalUnits): self
-    {
-        $this->rentalUnits = $rentalUnits;
-
-        return $this;
-    }
-
-    /**
-     * @param RentalUnit[] $rentalUnits
+     * @param RentalUnit $rentalUnit
      * @return self
      */
-    public function addRentalUnit(RentalUnit $rentalUnit): self
+    private function addRentalUnit(RentalUnit $rentalUnit): self
```

Comment:
Zie ik het nu verkeerd of kan deze nu nergens gebruikt worden? Want hij is private maar wordt in de class zelf nergens aangeroepen?

Reply:
- [x] uitleg geven over beperking serializer (nodig om serializer te vertellen welk object hij moet gebruiken in geval van arrays